systemvm/debian/opt/cloud/bin/configure.py

```diff
@@ -930,8 +936,20 @@ def forward_vpc(self, rule):
         if not rule["internal_ports"] == "any":
             fw_output_rule += ":" + self.portsToString(rule["internal_ports"], "-")
 
+        fw_postrout_rule2 = "-j SNAT --to-source %s -A POSTROUTING -s %s -d %s/32 -o %s -p %s -m %s --dport %s" % \
+              (
```

Comment:
@ustcweizhou can you fix the indent issue here, run `pycodestyle --max-line-length=179` on this file to confirm. See Travis' 1st job failure for hints: `../debian/opt/cloud/bin/configure.py:940:15: E127 continuation line over-indented for visual indent`

Reply:
@rhtyd 
checks should be ok now.